srtcore/sync_cxx11.cpp

```diff
@@ -16,6 +16,34 @@
 #include "srt_compat.h"
 #include "common.h"
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
Better put this into an anonymous namespace to avoid name clash risk.

Reply:
Done.